WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -109,8 +111,13 @@
     private static Context mContext;
     private static BitmapLruCache mBitmapCache;
 
+    protected void attachBaseContext(Context base) {
```

Comment:
This is already handled by `MultiDexApplication` (in fact, it's the only thing `MultiDexApplication` adds to `Application`).